src/main/java/com/code_intelligence/jazzer/Jazzer.java

```diff
@@ -321,6 +411,10 @@ private static boolean isPosix() {
     return FileSystems.getDefault().supportedFileAttributeViews().contains("posix");
   }
 
+  private static boolean isAndroid() {
+    return Boolean.parseBoolean(System.getProperty("jazzer.android", "false"));
```

Comment:
Just to be safe, could we add `!isAndroid()` to `isPosix()` in case the latter currently returns `true` on Android?